### DISCUSSION TITLE
Express Serverless: The default export must be a function or server.

### DISCUSSION THREAD

**Discussion by ajt001** [vercel/community#5630] [Help]
### Summary

Here are two almost same code snippets.
I have an issue where the balances (first snippet) does not work. Error is:
```
Invalid export found in module "../../api/balances.js".
The default export must be a function or server.
INIT_REPORT Init Duration: 859.67 ms	Phase: init	Status: error	Error Type: Runtime.ExitError
```
What's interesting is the balances was working, then breaks. 

<img width="1101" alt="image" src="https://github.com/vercel/community/assets/39484953/b1cd5474-601d-4fec-a1c2-7ed130bf346f">
<img width="1101" alt="image" src="https://github.com/vercel/community/assets/39484953/de1e3c67-52e0-4ea9-bc21-e4e940994a57">

### Steps to Reproduce

Screenshots above.

**Comment by mikececco** [comment]
Did you manage to solve it?

**Reply by ajt001** [reply]
I can't recall how, but worked around it somehow so it is operating.

**Reply by mikececco** [reply]
You have the repo I can look into?
